### ISSUE TITLE
[SOLVED] IndexError: list assignment index out of range

### ISSUE THREAD

**Issue by presk0** [marph91/jimmy#24]
Failed :
[PYI-10033:ERROR] Failed to execute script 'jimmy_cli' due to unhandled exception!

```
=> uname -a
Linux domaiName 6.1.0-25-amd64 #1 SMP PREEMPT_DYNAMIC Debian 6.1.106-3 (2024-08-26) x86_64 GNU/Linux

=> ./jimmy-cli-linux --format cherrytree migration.ctd 
[11/24/24 03:03:53] INFO     Jimmy 0.0.34 (Pandoc 3.5)                    
                    INFO     Importing notes from "migration.ctd"         
                    INFO     Start parsing                                
                    WARNING  ignoring scale=small                         
Traceback (most recent call last):
  File "jimmy_cli.py", line 139, in <module>
  File "jimmy_cli.py", line 135, in main
  File "jimmy.py", line 123, in jimmy
  File "jimmy.py", line 80, in convert_all_inputs
  File "converter.py", line 66, in convert_multiple
  File "formats/cherrytree.py", line 261, in convert
  File "formats/cherrytree.py", line 194, in convert_to_markdown
  File "formats/cherrytree.py", line 194, in convert_to_markdown
  File "formats/cherrytree.py", line 194, in convert_to_markdown
  [Previous line repeated 1 more time]
  File "formats/cherrytree.py", line 197, in convert_to_markdown
IndexError: list assignment index out of range
[PYI-10033:ERROR] Failed to execute script 'jimmy_cli' due to unhandled exception!

241124-03:10:18-presko@domaiName-~/Documents
=> ll migration.ctd 
-rw-r--r-- 1 presko presko 547411 Nov 24 02:12 migration.ctd

241124-03:10:30-presko@domaiName-~/Documents
=> file migration.ctd 
migration.ctd: XML 1.0 document, Unicode text, UTF-8 text, with very long lines (21991)
```

The file was made from "save as" cherrytree file format XML

Regards, and thanks for your work

**Comment by presk0** [comment]
And by the way :  why/how a python trace on a binary file ?

file jimmy-cli-linux 
jimmy-cli-linux: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, BuildID[sha1]=04804d3c31218f938502cbed5cdd1af09d59a8f0, for GNU/Linux 2.6.32, stripped

**Comment by marph91** [comment]
It's an implementation error. I made the affected part of the code more robust. But it's not obvious how this error was caused. So feel free to share the cherrytree file if you want.

I will provide a release with the fix later today.

> And by the way : why/how a python trace on a binary file ?

The binary file is basically a python environment that's executing the script. It's packaged by [pyinstaller](https://pyinstaller.org/). I simply like coding in python and feel more productive than using any other programming language :)

**Comment by marph91** [comment]
> File "formats/cherrytree.py", line 197, in convert_to_markdown
IndexError: list assignment index out of range

Should be fixed in [v0.0.35](https://github.com/marph91/jimmy/releases/tag/v0.0.35). Can you try it?

**Comment by presk0** [comment]
That worked, congrats
The file is quite personnal, I will not share it if there is no need.
Thank you for your help and regards
